usr.sbin/sysinst/msg.mi.pt

```diff
@@ -0,0 +1,1528 @@
+discos.)
+}
+
+message mount_failed
+{A montagem de %s falhou. Continuar?
+}
+
+message nodisk
+{Nao foi encontrado nenhum disco r�gido para uso pelo NetBSD. Voc� ser�
+retornado ao menu original.
+}
+
+message onedisk
+{Encontrei apenas um disco, %s.
+Portanto, presumo que voc� queira %s NetBSD nele.
+}
+
+message ask_disk
+{Em qual disco voc� deseja %s NetBSD? }
+
+message Available_disks
+{Discos dispon�veis}
+
+message Available_wedges	{"wedges" existentes}
+
+message heads
+{cabe�as}
+
+message sectors
+{setores}
+
+message mountpoint
+{ponto de montagem (ou '-')}
+
+message cylname
+{cil}
+
+message secname
+{sec}
+
+message megname
+{MB}
+
+message gigname
+{GB}
+
+/* Called with:				Example
+ *  $0 = device name			wd0
+ *  $1 = partitioning scheme name	Guid Partition Table
+ *  $2 = short version of $1		GPT
+ *  $3 = disk size for NetBSD		3TB
+ *  $4 = full install size min.		127M
+ *  $5 = install with X min.		427M
+ */
+message	layout_prologue_none
+{Voc� pode usar um editor simples para definir os tamanhos das parti�oes do NetBSD,
+ou aplicar os tamanhos e conte�dos de parti�ao padrao.}
+
+/* Called with:				Example
+ *  $0 = device name			wd0
+ *  $1 = partitioning scheme name	Guid Partition Table
+ *  $2 = short version of $1		GPT
+ *  $3 = disk size for NetBSD		3TB
+ *  $4 = full install size min.		127M
+ *  $5 = install with X min.		427M
+ */
+
+message	layout_prologue_existing
+{Se voc� nao quiser usar as parti�oes existentes, voc� pode
+usar um editor simples para definir os tamanhos das parti�oes NetBSD,
+ou remover as existentes e aplicar os tamanhos de parti�ao padrao.}
+
+/* Called with:				Example
+ *  $0 = device name			wd0
+ *  $1 = partitioning scheme name	Guid Partition Table
+ *  $2 = short version of $1		GPT
+ *  $3 = disk size for NetBSD		3TB
+ *  $4 = full install size min.		127M
+ *  $5 = install with X min.		427M
+ */
+message layout_main
+{
+Voc� ter� entao a oportunidade de alterar qualquer um dos detalhes da parti�ao
+
+A parte NetBSD (ou livre) do seu disco ($0) custa $3.
+
+Uma instala�ao completa requer pelo menos $4 sem X e
+pelo menos $5 se os conjuntos X estiverem inclu�dos.}
+
+message Choose_your_size_specifier
+{Escolher mega ou gigabytes dar� tamanhos de parti�ao pr�ximos
+� sua escolha, mas alinhados aos limites do cilindro.
+Escolher setores permitir� que voc� especifique
+os tamanhos com mais precisao. Na maioria dos discos, h� pouco a ganhar com
+alinhamento de cilindro. Em discos muito antigos, � mais eficiente
+escolher tamanhos de parti�ao que sejam m�ltiplos exatos do seu tamanho real
+de cilindro.
+
+Escolha seu especificador de tamanho}
+
+message ptnsizes
+{Agora voc� pode alterar os tamanhos das parti�oes do sistema. O padrao �
+alocar todo o espa�o para o sistema de arquivos raiz. No entanto, voc� pode desejar
+ter sistemas de arquivos /usr (arquivos de sistema adicionais), /var (arquivos de log etc.)
+ou /home (diret�rios home dos usu�rios) separados.
```

Comment:
I suspect this "home"  in the parenthesis is not really helping much with the semantics?

Reply:
How would you translate home directory ?
Personally this is part of the computer guy's jargon, but a good translation is always welcome.

Reply:
I'm completely OK with home. My point was more in the sense of the extra remark being a bit redundant.